File: src/main/java/aurora/jwt/common/dto/Preferences.java
package aurora.jwt.common.dto;

public class Preferences {

    private String locale;
    private String timezone;
    private String fileEncoding;

    public Preferences() {}
    public Preferences(String locale, String timezone, String fileEncoding) {
        this.locale = locale;
        this.timezone = timezone;
        this.fileEncoding = fileEncoding;
    }

    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getTimezone() {
        return timezone;
    }

    public void setTimezone(String timezone) {
        this.timezone = timezone;
    }

    public String getFileEncoding() {
        return fileEncoding;
    }

    public void setFileEncoding(String fileEncoding) {
        this.fileEncoding = fileEncoding;
    }
}

File: src/main/java/aurora/jwt/common/dto/Preferences.kt
package aurora.jwt.common.dto

class Preferences {
    var locale: String? = null
    var timezone: String? = null
    var fileEncoding: String? = null

    constructor() {}
    constructor(locale: String?, timezone: String?, fileEncoding: String?) {
        this.locale = locale
        this.timezone = timezone
        this.fileEncoding = fileEncoding
    }
}
